example/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "RNNHooksSample",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "test": "jest",
+    "lint": "eslint ."
+  },
+  "dependencies": {
+    "prop-types": "^15.7.2",
```

Comment:
I'd set this as "15.7.2" also we could set `save-exact=true` in the `.npmrc` file